.github/workflows/ci.yml

```diff
@@ -38,7 +39,7 @@ jobs:
           path: '**/node_modules'
           key: ${{ runner.os }}-test-modules-${{ hashFiles('**/yarn.lock') }}
       - name: Install dependencies
-        run: yarn install
+        run: yarn install  --frozen-lockfile --ignore-engines
```

Comment:
Could you also mention this in the README, like here? https://github.com/rubensworks/jbr.js/commit/2785b17d5897243f342e7383824c987218078bf0

Reply:
There does not seem to be a development setup section in the readme at all, so it does not have any mention of installing dependencies for this project such as the one linked in the jbr readme. There is only a global install example of the final published package for users of it.

Do I just skip this suggestion? Adding an entire development setup section to the readme feels a bit excessive, especially since the readme as a whole is more aimed towards using the published package.

Reply:
But the global installations will also require this flag, right?

Reply:
That is true! I had to test it, since I never tried global installs. Will fix. Thank you.

Reply:
The flag is now in the readme, in the global install command example. It does not have any explanation, but the jbr one also did not seem to have any. Hopefully that is enough.